Core/Inc/bmb.h

```diff
@@ -114,13 +121,18 @@ bool initASCI(uint32_t *numBmbs);
 void initBmbs(uint32_t numBmbs);
 
 // TODO update description
-void setGpio(uint32_t numBmbs, uint16_t gpioSetting);
+void cyclicUpdateBMBData(Bmb_S* bmb, uint32_t numBmbs);
```

Comment:
Idk if the cyclic is required here. It's technically correct but maybe a bit distracting. Probably simpler to just call it updateBmbData

Reply:
I like calling it cyclic because it is not a base function of the bmb like updateVoltage or temp, it is a special function used by main task. Imo, the presence of the nonblocking timer code really makes it more than just updateBmbData.